docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  calypso:
+    image: node:6.10.0
+    extra_hosts:
+      - 'calypso.localhost:0.0.0.0'
+    environment:
+      CALYPSO_ENV: 'development'
+      NODE_ENV: 'development'
+      HOST: 'calypso.localhost'
+      PORT: 3000
+    ports:
+      - '3000:3000'
```

Comment:
oh, Docker and it's crazy perception on how things should work ...

Another point though, I don't think it's wise to assume that we even know 127.0.0.1 will even be the loopback device. It might only be ::1 in an ipv6-only environment. Docker may not be bound to the lo device as well, which is something I do in production for my blog. I've also been on machines where there's no host entry for localhost, so that's another potentially false assumption. I don't want to this to be one of those products that assumes people's network architectures and then breaks when you need it to "just work" in a stranger environment.

I don't want to assume someone's network architecture and then have to deal with issues because of it. I think this is something that's perfectly acceptable to run exposed, since it will also be exposed in real life, to a much larger number of potentially malicious users. I think adding a note about this is enough. I believe `make run` will expose the port to the world as well, otherwise this container wouldn't work at all...